vshard/replicaset.lua

```diff
@@ -577,13 +577,33 @@ local function replicaset_template_multicallro(prefer_replica, balance)
     local function pick_next_replica(replicaset, now)
         local r
         local master = replicaset.master
+
         if balance then
+            local prefered_zone = replicaset.priority_list[1].zone
             local i = #replicaset.priority_list
             while i > 0 do
                 r = replicaset_balance_replica(replicaset)
                 i = i - 1
                 if r:is_connected() and (not prefer_replica or r ~= master) and
-                   replica_check_backoff(r, now) then
+                    replica_check_backoff(r, now)
+                then
```

Comment:
Please, revert unnecessary changes like these. These 3 lines didn't have to change at all. Besides, we never put `then` on a separate line. Lets keep it consistent with our code style.